sdk/template/template/test/public/configurationClient.spec.ts

```diff
@@ -90,6 +90,7 @@ describe("[AAD] ConfigurationClient functional tests", function () {
     // the `getConfigurationSetting` method is being traced correctly, that the
     // tracing span is properly parented and closed.
     it("supports tracing", async () => {
+      await recorder.setMatcher("HeaderlessMatcher");
```

Comment:
```suggestion
      // Playback fails in the browser without the "HeaderlessMatcher" 
      // 
      // If-Modified-Since & If-None-Match headers are not present in the recording and the playback has these headers
      // Proxy tool doesn't treat these headers differently, tries to match with the recording, and fails.
      // More details here - https://github.com/Azure/azure-sdk-tools/issues/2674
      await recorder.setMatcher("HeaderlessMatcher");
```